vscode_remote_tunnel/start_vscode_tunnel.py

```diff
@@ -34,22 +34,23 @@ def create_dirpath(arg):
     return dpath
 
 
-def write_script(cpu, mem_per_cpu, partition, logdir, tunnel_name, include_node, exclude_node):
+def write_script(cpu, gpu, mem, partition, logdir, tunnel_name, include_node, exclude_node):
     "constructs slurm job script and writes to file"
 
     Path(logdir).mkdir(parents=True, exist_ok=True)
 
     if not tunnel_name:
         tunnel_name = "${SLURMD_NODENAME}_cheaha_tunnel"
-    
+
     newline = "\n"  # newline to use inside fstring (https://stackoverflow.com/a/44780467/3998252)
     script_txt = f"""\
         #!/bin/bash
         #
         #SBATCH --job-name=vscode_tunnel
         #SBATCH --ntasks=1
         #SBATCH --cpus-per-task={cpu}
-        #SBATCH --mem={mem_per_cpu}
+        #SBATCH --gres=gpu:{gpu}
```

Comment:
Any reason `--gres` was chosen over `--gpus`?

Reply:
That is how the RC docs suggest for requesting GPUs

<img width="1232" alt="image" src="https://github.com/user-attachments/assets/bfd6be28-4137-40d7-8d15-87d809ee9706">